pulpcore/app/settings.py

```diff
@@ -489,27 +489,59 @@ def otel_middleware_hook(settings):
     return data
 
 
+def load_plugin_config_hook(settings):
+    # Enumerate the installed Pulp plugins during the loading process for use in the status API
+    ENABLED_PLUGINS = getattr(settings, "ENABLED_PLUGINS", None)
+    installed_plugins = []
+    installed_plugin_apps = []
+
+    for entry_point in entry_points(group="pulpcore.plugin"):
+        if ENABLED_PLUGINS is not None and entry_point.name not in ENABLED_PLUGINS:
+            continue
+        installed_plugins.append(entry_point.module)
+        installed_plugin_apps.append(entry_point.load())
+
+    plugin_settings = Dynaconf(
+        PRELOAD_FOR_DYNACONF=[f"{module}.app.settings" for module in installed_plugins]
+    )
+
+    data = {"dynaconf_merge": True}
+    data.update(plugin_settings.as_dict())
+    data.update(settings.as_dict())
+    data["INSTALLED_APPS"].extend(installed_plugin_apps)
+    data["INSTALLED_APPS"].append("dynaconf_merge_unique")
+    return data
+
+
 settings = DjangoDynaconf(
     __name__,
     ENVVAR_PREFIX_FOR_DYNACONF="PULP",
     ENV_SWITCHER_FOR_DYNACONF="PULP_ENV",
-    PRELOAD_FOR_DYNACONF=[
-        "{}.app.settings".format(plugin_name) for plugin_name in INSTALLED_PULP_PLUGINS
-    ],
     ENVVAR_FOR_DYNACONF="PULP_SETTINGS",
     load_dotenv=False,
     validators=[
         api_root_validator,
         cache_validator,
+        enabled_plugins_validator,
         sha256_validator,
         storage_validator,
         unknown_algs_validator,
         json_header_auth_validator,
         authentication_json_header_openapi_security_scheme_validator,
     ],
-    post_hooks=otel_middleware_hook,
+    post_hooks=(otel_middleware_hook),
```

Comment:
```suggestion
    post_hooks=(otel_middleware_hook,),
```

Missing a comma? IIRC this must be a list or tuple